### ISSUE TITLE
Installation instructions refer to files which aren't present

### ISSUE THREAD

**Issue by nikolawannabe** [rackerlabs/openstack-guest-agents-unix#62]
6) ln -s /usr/share/nova-agent/<VERSION>/etc/nova-agent.init /etc/init.d/nova-agent

This file is present neither in that location (the location doesn't exist after install), nor in the source tree.  I assume it refers to the .in files.

**Comment by naterh** [comment]
Right.  Building the project results in a tab all when uncompressed has an install script that generates all these for you depending on the distribution and init system your installing onto.

**Comment by naterh** [comment]
*tarball
